docs/docs/guides/fonts.md

```diff
@@ -0,0 +1,124 @@
+# Fonts
+
+## Loading Google Fonts with next-font, expo-font, and Tailwind/Nativewind
+
+This guide was written using the expo-router starter in mind. 
+
+1. Load fonts in expo, in apps/expo/_layout.tsx. Make sure to `npx expo add @expo-google-fonts/roboto-slab` whatever fonts you want to use.
```

Comment:
I think we could make this more expo-router-agnostic.

Create this file, and then wrap your `App.tsx` with it.

If you're using Expo Router, then it would go in the `app/_layout.tsx` file (and show a very minimal example).

For example, the way `Fonts` is used in the [Dripsy guide](https://www.dripsy.xyz/fonts#2-loading-in-fonts-with-expo-font-or-whatever-loader-you-prefer) in a separate file and then imported simply.

Reply:
Also, the instructions should work for custom font files themselves